app/src/main/java/org/sopt/and/data/mapper/todata/UserMapper.kt

```diff
@@ -0,0 +1,23 @@
+package org.sopt.and.data.mapper.todata
+
+import org.sopt.and.data.remote.model.request.LoginRequestDto
+import org.sopt.and.data.remote.model.request.UserInfoUpdateRequestDto
+import org.sopt.and.data.remote.model.request.UserRegistrationRequestDto
+import org.sopt.and.domain.model.User
+
+fun User.toUserRegistrationRequestDto(): UserRegistrationRequestDto = UserRegistrationRequestDto(
+    username = this.username,
+    password = this.password,
+    hobby = this.hobby
+)
+
+fun User.toLoginRequestDto(): LoginRequestDto = LoginRequestDto(
+    username = this.username,
+    password = this.password
+)
+
+
+fun User.toUserInfoUpdateRequestDto(): UserInfoUpdateRequestDto = UserInfoUpdateRequestDto(
+    hobby = this.hobby,
+    password = this.password
+)
```

Comment:
더 코틀린스러운 코드로 만들자면,
- 멤버 접근 시 기본적으로 this를 생략할 수 있습니다
- 명시적인 리턴 타입이 없어도 충분히 코드가 명확한 함수입니다 (동일한 네이밍의 Dto가 3번 반복됨)
```suggestion
fun User.toUserInfoUpdateRequestDto() = UserInfoUpdateRequestDto(
    hobby = hobby,
    password = password
)
```

코드 취향일 뿐이라 그대로 하셔도 됩니다 ~~